src/components/CohortBuilder/Summary/index.js

```diff
@@ -72,10 +71,18 @@ const multiHeader = (
 );
 
 const Summary = ({ theme, sqon, api }) => (
-  <QueriesResolver api={api} sqon={sqon} queries={[demographicQuery({ ...{ sqon } })]}>
-    {({ loading, data }) => {
-      return loading || !data ? (
-        <div> loading</div>
+  <QueriesResolver api={api} queries={[demographicQuery(sqon), demographicQuery(sqon)]}>
+    {({ isLoading, data }) => {
+      const [demographicData] = data || [];
+
+      return isLoading ? (
+        <Row nogutter>
+          <div className={theme.fillCenter} style={{ marginTop: '30px' }}>
+            <LoadingSpinner color={theme.greyScale11} size={'50px'} />
+          </div>
+        </Row>
```

Comment:
Just food for thought for later, I feel it's more fitting to handle loading state in each card :) (or even no loading state at all like the file repo)

Reply:
Does one request not mean one loading state?